LICENSE-3rdparty.csv

```diff
@@ -42,9 +42,10 @@ import,androidx.vectordrawable,Apache-2.0,Copyright 2018 The Android Open Source
 import,androidx.versionedparcelable,Apache-2.0,Copyright 2018 The Android Open Source Project
 import,androidx.viewpager,Apache-2.0,Copyright 2018 The Android Open Source Project
 import,androidx.work,Apache-2.0,Copyright 2018 The Android Open Source Project
+import,com.android.tools,GNU General Public License, Version 3, 29 June 2007. Copyright 2024 Datadog, Inc.
```

Comment:
probably everything under `com.android.tools` is under Apache-2.0 license by clicking different items under https://mvnrepository.com/artifact/com.android.tools. So the same license as above (and same Copyright)

Reply:
Yeah some mention GNU some Apache. I will add Apache instead.